themes/classic/modules/ps_contactinfo/ps_contactinfo.tpl

```diff
@@ -56,7 +56,7 @@
         <br>
         {* [1][/1] is for a HTML tag. *}
         {l
-          s='Email us: [1]%email%[/1]'
+          s='Email us: <a href="mailto:%email%" class="dropdown">[1]%email%[/1]</a>'
```

Comment:
No HTML tags into translated strings.
Please, make the same as the `<span>` tag.